Fix Custom Vision Service tutorial for Python

c-w opened the pull request:
### Fix de-reference of ids in example code

For the current version of this library (0.1.0), the example code fails
with an attribute error because the `id` property does not exist on the
`Tag` and `Project` classes. Changing the references to `Id` fixes the
error.

### Make import style consistent

The rest of the example snippets in this tutorial directly import the
objects required by the code instead of importing their module. This
changes makes the import style consistent by directly importing the
`TrainingApi` object instead of the `training_api` module.

azurecla commented:
Hi __@ErikCL__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

sethmanheim commented:
Hi @ErikCL, thanks for catching these; they happened during the conversion from MSDN to Markdown. We appreciate the contribution! @tysonn, OK to merge.

ErikCL commented:
@sethmanheim sure thing. I'm glad I could help. 

FYI: There were some other formatting problems with the code blocks on the lower half of that page that were probably due to another conversion glitch. I spent a little bit of time trying to fix them unsuccessfully as well. If they haven't been fixed by someone else you might want to try to fix them too. There is probably a hidden character or something breaking the markdown, but I couldn't figure it out.

sethmanheim commented:
@ErikCL Thanks Erik, I definitely see the formatting issues you mentioned. I'll fix them ASAP, and thanks again!

sethmanheim commented:
@tysonn You can close this pull request if you want, because I incorporated Erik's changes into a larger PR that also addressed his follow-up feedback. Thanks!

c-w committed:
Make import style consistent

The rest of the example snippets in this tutorial directly import the
objects required by the code instead of importing their module. This
changes makes the import style consistent by directly importing the
`TrainingApi` object instead of the `training_api` module.

c-w committed:
Fix de-reference of ids in example code

For the current version of this library (0.1.0), the example code fails
with an attribute error because the `id` property does not exist on the
`Tag` and `Project` classes. Changing the references to `Id` fixes the
error.
